web/package.json

```diff
@@ -43,11 +43,8 @@
     "eslint-plugin-prettier": "^5.2.3",
     "postcss": "^8",
     "prettier": "^3.5.2",
+    "prettier-plugin-tailwindcss": "^0.6.14",
     "tailwindcss": "^4.1.11",
     "typescript": "^5"
-  },
-  "overrides": {
-    "@types/react": "19.1.6",
-    "@types/react-dom": "19.1.6"
```

Comment:
you shouldn't need these anymore. no not related to tailwind but something you could clean up